lib/atlas/scaler/time_curve_scaler.rb

```diff
@@ -16,20 +16,19 @@ def self.call(*args)
 
     private_class_method :new
 
-    def initialize(base_dataset, scaling_factor, derived_dataset)
-      @base_dataset = base_dataset
-      @scaling_factor = scaling_factor
+    def initialize(base_dataset, derived_dataset)
+      @base_dataset    = base_dataset
       @derived_dataset = derived_dataset
+      @scaling_factor  = derived_dataset.scaling.factor
     end
 
     # Public: Scales the curves and saves them to new csv files
     #
-    # Returns nil
+    # Returns an array
```

Comment:
> I don't think a user of this class would expect this

In it's [current state](https://github.com/quintel/atlas/blob/master/lib/atlas/scaler.rb#L25) it shouldn't have to return anything i.e. it doesn't matter to the application what it returns.

> I think it should be the scaled curves from the derived dataset.

This makes more sense generally speaking than returning either `nil` or the `time_curves`.